examples/hid_mouse/src/main.c

```diff
@@ -168,6 +169,32 @@ int read_from_flash(void *db, const int max_size)
     return KV_OK;
 }
 
+#if (INGCHIPS_FAMILY == INGCHIPS_FAMILY_916)
+static void QDEC_PclkCfg(void)
+{
+    uint32_t hclk = SYSCTRL_GetHClk();
+    uint32_t qdecClk = SYSCTRL_GetClk(SYSCTRL_ITEM_APB_QDEC);
+    uint8_t div = hclk / qdecClk;
+    if (hclk % qdecClk)
+        div++;
+    if (!(div >> 4))
+        SYSCTRL_SetPClkDiv(div);
+}
+static void QDEC_Setup(void)
+{
+    QDEC_PclkCfg();
+    SYSCTRL_ClearClkGate(SYSCTRL_ITEM_APB_QDEC);
+    SYSCTRL_ReleaseBlock(SYSCTRL_ITEM_APB_PinCtrl |
+                         SYSCTRL_ITEM_APB_QDEC);
+    PINCTRL_SelQDECIn(21, 22);
+
+    SYSCTRL_SelectQDECClk(SYSCTRL_CLK_HCLK, 100);
```

Comment:
It does not make sense: after `QDEC_PclkCfg()`, `SYSCTRL_SelectQDECClk(SYSCTRL_CLK_HCLK, 100)`.